keps/NNNN-kep-template/kep.yaml

```diff
@@ -22,6 +22,8 @@ replaces:
   - "/keps/sig-ccc/3456-replaced-kep"
 
 # The target maturity stage in the current dev cycle for this KEP.
+# If the purpose of this KEP is to deprecate a user-visible feature
```

Comment:
This is implemented in api/proposal.go type `Stage` and type `Milestone`.

We alrady have a milestone `Deprecated` and another `Removed`.  I think what we want to propose is that `stage` can be `deprecated` (meaning "deprecation was announced"), `deprecated-disabled (meaning "still present but off by default and soon to be removed"), and "removed", with corresponding `milestones` for each.

Reply:
Ah, you are right.

Reply:
modified